torch/csrc/jit/codegen/cuda/ir_graphviz.cpp

```diff
@@ -80,25 +80,7 @@ class IrNodeLabel : private OptInConstDispatch {
   }
 
   void handle(const IterDomain* id) override {
-    if (id->isReduction()) {
-      label_ << "r";
-    } else {
-      label_ << "i";
-    }
-
-    switch (id->parallel_method()) {
-      case (ParallelType::Vectorize):
-        label_ << "V";
-        break;
-      case (ParallelType::Unroll):
-        label_ << "U";
-        break;
-      case (ParallelType::Serial):
-        label_ << "S";
-        break;
-      default:
-        label_ << id->parallel_method();
-    }
+    label_ << id->getIterType();
```

Comment:
Is this intended to not printout ParallelType any more?

Reply:
Good catch, will fix.